### PR TITLE
Update related_website_sets.JSON

### PR DESCRIPTION
**Pull request by abatahi**

### CODE REVIEW

**Review by shuranhuang — APPROVED** [review]
LGTM

**Comment by jc23e** [comment]
Hola buenas tardes como está el conocimiento

File: related_website_sets.JSON [hunk 1]
```diff
@@ -738,6 +738,16 @@
...
     }
  },
+ {
+    "contact": "abatahi@nvidia.com",
+    "primary": "https://nvidia.com",
+    "associatedSites": [
+      "https://geforcenow.com"
+    ],
+    "rationaleBySite": {
+      "https://geforcenow.com": "A site that represents the WEB client for NVIDIA GEFORCE NOW service. Players are able to instantly stream their PC game library. NVIDIA branding is clearly visible on this site."
```
Comment (Parasos) [inline related_website_sets.JSON:748]: related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
4fe104ef461bc17146db6acc5f1809b705d92712

File: related_website_sets.JSON (hunk 1 above)
Comment (Danaifarm) [reply related_website_sets.JSON:748]: - []()Vito

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:748]: > related_website_sets.JSON

ab6bf8ae993a0488831f1d862ac28405518b2457

**Review by ratchanonsuttawas — COMMENTED** [review]
ab6bf8ae993a0488831f1d862ac28405518b2457

**Review by ratchanonsuttawas — COMMENTED** [review]
Changes

**Comment by dvnprtamyd** [comment]
>

**Pull request merged**